dashboard/src/t5gweb/templates/ui/updates.html

```diff
@@ -52,15 +52,24 @@
                                  <span class="anchor" id={{ new_comments[account][status][details]['case_number'] }}></span>
                                  <ul class=highlight>
                                     {% if new_comments[account][status][details]['bugzilla'] != "None" %}
-                                    <li class="header" id = {{new_comments[account][status][details]['case_number'] }}><a href=https://access.redhat.com/support/cases/#/case/{{new_comments[account][status][details]['case_number'] }} target="_blank">{{new_comments[account][status][details]['case_number'] }}</a> | <span class="severity">Severity: {{ new_comments[account][status][details]['severity'] }}</span> | <span class="case">{{new_comments[account][status][details]['summary']}}</span> (Jira: <a href=https://issues.redhat.com/browse/{{ details }} target="_blank">{{ details }}</a> / {{new_comments[account][status][details]['assignee']['displayName'] }} / Bugzilla: <a href=https://bugzilla.redhat.com/show_bug.cgi?id={{new_comments[account][status][details]['bugzilla'] }} target="_blank">{{new_comments[account][status][details]['bugzilla'] }}</a>)</li>
+                                    <li class="header" id = {{new_comments[account][status][details]['case_number'] }}><a href=https://access.redhat.com/support/cases/#/case/{{new_comments[account][status][details]['case_number'] }} target="_blank">{{new_comments[account][status][details]['case_number'] }}</a> | <span class="severity">Severity: {{ new_comments[account][status][details]['severity'] }}</span> | <span class="case">{{new_comments[account][status][details]['summary']}}</span> (Jira: <a href=https://issues.redhat.com/browse/{{ details }} target="_blank">{{ details }}</a> / {{new_comments[account][status][details]['assignee']['displayName'] }}) </li>
```

Comment:
one bit of feedback from a colleague: can you tweak it slightly so the assignee is the last item? So  (Jira ... / Bugzilla ... / Whoever)

Reply:
The bugzilla lists were getting kind of long, so I added them to a separate bullet:
- Card info .... (Jira ... / Assignee)
     - [Bugzilla 1 - Target Release: .... - BZ Assignee: ....] [Bugzilla 2 - Target Release: .... - BZ Assignee: ....] etc. 
                - Comments ....
Should we change this?